packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -121,13 +121,54 @@ class AndroidDomainError extends DomainError {
 
 class IosDomainError extends DomainError {
   const IosDomainError(super.title, super.explanation, super.fixDetails);
-  // TODO: Add  domain errors for iOS.
+  // TODO(hangyujin): Finalize strings for these domain errors.
 
   /// Existence of an Apple-App-Site-Association file.
   static const existence = IosDomainError(
     'Apple-App-Site-Association file does not exist',
-    '',
-    '',
+    'This test checks whether the Apple-App-Site-Association file, '
+        'which is used to verify the association between the app and the '
+        'domain name, exists under your domain.',
+    'Add a Digital Apple-App-Site-Association file to all of the '
+        'failed website domains',
+  );
+
+  /// AASA file should define a link to this app.
+  static const appIdentifier = IosDomainError(
+    'App identifier not found',
+    'The test checks your Digital Apple-App-Site-Association file '
+        'for App identifier validation, which the mobile device '
+        'uses to verify ownership of the app.',
```

Comment:
'The Digital Apple-App-Site-Association file requires an app identifier so that the mobile device can verify ownership of the app.'